src/common/playlists/PlayHeaderActions.jsx

```diff
@@ -31,17 +31,19 @@ const PlayHeaderActions = ({ play }) => {
           <span className="sr-only">GitHub</span>
         </a>
       )}
-      {play.blog && (
-        <a
-          target="_blank"
-          rel="noopener noreferrer"
-          className="action"
-          href={play.blog}
-        >
-          <AiOutlineRead className="icon" size="24px" />
-          <span className="sr-only">Blog</span>
-        </a>
-      )}
+      <a
+        target="_blank"
+        rel="noopener noreferrer"
+        className="action"
+        href={
+          play.blog
+            ? play.blog
+            : `https://github.com/atapas/react-play/tree/main/src${play.path}/Readme.md`
```

Comment:
The github location has been changed.

Please use the correct URL

`https://github.com/reactplay/react-play/tree/main/src${play.path}/Readme.md`

Reply:
Done